base/pkg/entry.jl

```diff
@@ -719,7 +720,7 @@ function test!(pkg::AbstractString,
                     --compilecache=$(Bool(Base.JLOptions().use_compilecache) ? "yes" : "no")
                     --check-bounds=yes
                     --startup-file=$(Base.JLOptions().startupfile != 2 ? "yes" : "no")
-                    $test_path
+                    --eval $code
```

Comment:
is running the runtests.jl inside a module via evalfile going to result in any packages' tests behaving differently?